frontend/components/AddressForm/index.jsx

```diff
@@ -64,7 +65,7 @@ export class AddressForm extends Component {
   constructor(props) {
     super(props);
 
-    this.countriesList = countriesList();
+    this.countriesList = countriesList(props.config.countryCodes);
```

Comment:
I guess this won't be needed anymore, since the `FormBuilder`s `country` element already has a `countries` attribute.